flake.nix

```diff
@@ -471,6 +471,8 @@
               pkgs.vault
               pkgs.wit-deps
               pkgs.go
+              pkgs.kubernetes-helm
```

Comment:
Noticed these were missing and I use the devShell. Added them for anyone else who might be doing dev work in the same way